linux_os/guide/system/apparmor/all_apparmor_profiles_in_enforce_complain_mode/tests/correct_all_apparmor_profiles_in_complain.pass.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+# platform: package[apparmor]
```

Comment:
This actually doesn't work this way. I admit its confusing. But automatus is different than format of rule.ymls. You should use `packages`.

```suggestion
# packages = apparmor
```

Reply:
Hi @teacup-on-rockingchair this actually occurs there multiple times, please replaces all occurrences.